Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,31 @@
 import React from 'react';
 import './App.css';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import ActionButton from './components/ButtonComponents/ActionButton';
+import LongButton from './components/ButtonComponents/LongButton';
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="cont">
+    <CalculatorDisplay />
+      <LongButton text="clear" />
```

Comment:
At this point you should notice that you're writing a lot of the same things over and over again and only changing one value. Think about what you could do instead of this that would allow you to write this only once. Think about why you would want to use this higher order function besides the fact that it would allow you to write less and neater code.. This will be important as you move on to state.